server/odc-service/src/main/java/com/oceanbase/odc/service/flow/factory/FlowResponseMapperFactory.java

```diff
@@ -305,35 +306,31 @@ private FlowInstanceMapper generateMapper(@NonNull Collection<Long> flowInstance
         /**
          * Get Database associated with each TaskEntity
          */
-        Set<Long> databaseIds = taskId2TaskEntity.values().stream().filter(entity -> entity.getDatabaseId() != null)
```

Comment:
why should you change these codes? is there any merging problem during the conflict solving?

Reply:
Yes, there is a bug of parameter validate failed. Because some type of task has no databaseId.